multi_label_trainer_tfidf.ipynb

```diff
@@ -4,10 +4,9 @@
   "metadata": {
```

Comment:
<p>For an end-to-end demonstration of classification with imbablanced data, refer to <a href="https://keras.io/examples/structured_data/imbalanced_classification/" target="_blank">Imbalanced classification: credit card fraud detection</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>